src/nplinker/logconfig.py

```diff
@@ -14,20 +14,21 @@
 
 import logging
 import sys
+from typing_extensions import Self
 
 
 class LogConfig():
 
-    active_loggers = {}
+    active_loggers: dict[Self, logging.Logger] = {}
     logfmt = '%(asctime)s [%(levelname)s] %(filename)s:%(lineno)d, %(message)s'
     default_loglevel = logging.INFO
     # default destination for new Loggers
     default_logdest = logging.StreamHandler(sys.stdout)
     # additional destinations to be added to new Loggers
-    additional_logdests = []
+    additional_logdests: list[logging.Logger] = []
```

Comment:
the type should be `list[logging.Handler]`

Reply:
See https://github.com/NPLinker/nplinker/pull/91/commits/190c433171355a1bd7607caba32397135292d7eb